### PR TITLE
Update iOS sensors

### PR DESCRIPTION
**Pull request by bgoncal**

### CODE REVIEW

File: docs/core/sensors.md [hunk 1]
```diff
@@ -45,13 +45,17 @@ On macOS, sensors update in the same situations as above as well as immediately
 | Sensor | Attributes | Description |
 | --------- | --------- | ----------- |
 | `sensor.activity` | `confidence`, `types` | The current activity type as computed by iOS. Requires motion permissions to be enabled. |
+| `sensor.app_version` | None | The current Home Assistant companion App for iOS App version. |
```
Comment (c0ffeeca7) [inline docs/core/sensors.md:48]: Is this the cersion of the HA companion app? to the iOS version?

Comment (bgoncal) [reply docs/core/sensors.md:48]: It's the "HA iOS App" app version :P

File: docs/core/sensors.md [hunk 2]
```diff
@@ -45,13 +45,17 @@ On macOS, sensors update in the same situations as above as well as immediately
...
 | `sensor.average_active_pace` | None | The averaged pace calculated by iOS from pedometer data. Units: meters per second, m/s |
 | `sensor.distance` | None | The estimated distance walked by the user since midnight local time. Units: meters, m |
 | `sensor.floors_ascended` | None | The approximate number of floors ascended by walking since midnight local time. |
 | `sensor.floors_descended` | None | The approximate number of floors descended by walking. Since |
+| `sensor.location_permission` | None | The current location permission selected by the user on location permission popup or modified in iOS settings. |
 | `sensor.sim_1` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.sim_2` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.steps` | None | The number of steps taken by the user. |
+| `sensor.watch_battery_level` | None | The battery level of 1 paired Apple Watch (Requires any Home Assistant complication installed on your watch face). |
+| `sensor.watch_battery_state` | None | The battery state of 1 paired Apple Watch (Requires any Home Assistant complication installed on your watch face). |
```
[the remark this answers is not in the document]
Comment (bgoncal) [reply docs/core/sensors.md:58]: Good point, I'll copy the iPhpne battery state itself:
```
The current charging state (either Charging, Not Charging, or Full) of the device.
```

File: docs/core/sensors.md [hunk 3]
```diff
@@ -45,13 +45,17 @@ On macOS, sensors update in the same situations as above as well as immediately
...
 | `sensor.distance` | None | The estimated distance walked by the user since midnight local time. Units: meters, m |
 | `sensor.floors_ascended` | None | The approximate number of floors ascended by walking since midnight local time. |
 | `sensor.floors_descended` | None | The approximate number of floors descended by walking. Since |
+| `sensor.location_permission` | None | The current location permission that was selected by the user. The permission can be set via the location permission popup or modified in the iOS settings. |
 | `sensor.sim_1` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.sim_2` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.steps` | None | The number of steps taken by the user. |
+| `sensor.watch_battery_level` | None | The battery level of 1 paired Apple Watch (Requires any Home Assistant complication installed on your watch face). |
+| `sensor.watch_battery_state` | None | The current charging state (either Charging, Not Charging, or Full) of 1 paired Apple Watch (Requires any Home Assistant complication installed on your watch face). |
```
Comment (c0ffeeca7) [inline docs/core/sensors.md:58]:
```suggestion
| `sensor.watch_battery_state` | None | The current charging state (either Charging, Not Charging, or Full) of 1 paired Apple Watch. Requires any Home Assistant complication installed on your watch face. |
```

File: docs/core/sensors.md [hunk 4]
```diff
@@ -45,13 +45,17 @@ On macOS, sensors update in the same situations as above as well as immediately
...
+| `sensor.app_version` | None | The current Home Assistant companion App for iOS App version. |
 | `sensor.average_active_pace` | None | The averaged pace calculated by iOS from pedometer data. Units: meters per second, m/s |
 | `sensor.distance` | None | The estimated distance walked by the user since midnight local time. Units: meters, m |
 | `sensor.floors_ascended` | None | The approximate number of floors ascended by walking since midnight local time. |
 | `sensor.floors_descended` | None | The approximate number of floors descended by walking. Since |
+| `sensor.location_permission` | None | The current location permission that was selected by the user. The permission can be set via the location permission popup or modified in the iOS settings. |
 | `sensor.sim_1` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.sim_2` | [See Below](#cellular-provider-sensor) | Name of your cellular provider. |
 | `sensor.steps` | None | The number of steps taken by the user. |
+| `sensor.watch_battery_level` | None | The battery level of 1 paired Apple Watch (Requires any Home Assistant complication installed on your watch face). |
```
Comment (c0ffeeca7) [inline docs/core/sensors.md:57]:
```suggestion
| `sensor.watch_battery_level` | None | The battery level of 1 paired Apple Watch. Requires any Home Assistant complication installed on your watch face. |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @bgoncal :+1:

File: docs/core/sensors.md (hunk 1 above)
Comment (c0ffeeca7) [inline docs/core/sensors.md:48]:
```suggestion
| `sensor.app_version` | None | The current **Home Assistant companion App for iOS** app version. |
```
Like this?

**Pull request merged**